--- avm/config-qca955x-1750_07.01
+++ freetz/config-qca955x-1750_07.01
@@ -1565,7 +1565,6 @@
 CONFIG_AVM_PAGE_TRACE=y
 CONFIG_AVM_NET_EVENT=y
 CONFIG_AVM_NET_TRACE=y
-CONFIG_UBIK2_ISDNSTACK_ON_CPU=0
 
 #
 # TFFS
@@ -2237,7 +2236,6 @@
 #
 # CONFIG_USB_MDC800 is not set
 # CONFIG_USBIP_CORE is not set
-# CONFIG_USB_MUSB_HDRC is not set
 # CONFIG_USB_DWC3 is not set
 # CONFIG_USB_DWC2 is not set
 # CONFIG_USB_CHIPIDEA is not set
