
--- setup.py
+++ setup.py
@@ -26,5 +26,12 @@
 
-from setuptools import setup, Extension, Feature
-from setuptools.command.build_ext import build_ext
-from setuptools import find_packages
+try:
+    from setuptools import setup, Extension, Feature
+    from setuptools.command.build_ext import build_ext
+    from setuptools import find_packages
+except ImportError:
+    from ez_setup import use_setuptools
+    use_setuptools("28.3.0")
+    from setuptools import setup, Extension, Feature
+    from setuptools.command.build_ext import build_ext
+    from setuptools import find_packages
 
