--- doc/example-config/dovecot.conf	2013-11-24 14:37:39.000000000 +0100
+++ doc/example-config/dovecot.conf	2011-03-27 00:00:00.000000000 +0100
@@ -22,30 +18,32 @@
 
 # Protocols we want to be serving.
 #protocols = imap pop3 lmtp submission
+protocols = imap
 
 # A comma separated list of IPs or hosts where to listen in for connections. 
 # "*" listens in all IPv4 interfaces, "::" listens in all IPv6 interfaces.
 # If you want to specify non-default ports or anything more complex,
 # edit conf.d/master.conf.
-#listen = *, ::
+listen = *, ::
 
 # Base directory where to store runtime data.
 #base_dir = /var/run/dovecot/
+base_dir = /var/run/dovecot/
 
 # Name of this instance. In multi-instance setup doveadm and other commands
 # can use -i <instance_name> to select which instance is used (an alternative
 # to -c <config_path>). The instance name is also added to Dovecot processes
 # in ps output.
 #instance_name = dovecot
 
 # Greeting message for clients.
-#login_greeting = Dovecot ready.
+login_greeting = Dovecot ready.
 
 # Space separated list of trusted network ranges. Connections from these
 # IPs are allowed to override their IP addresses and ports (for logging and
 # for authentication checks). disable_plaintext_auth is also ignored for
 # these networks. Typically you'd specify your IMAP proxy servers here.
-#login_trusted_networks =
+login_trusted_networks = 192.168.178.0
 
 # Space separated list of login access check sockets (e.g. tcpwrap)
 #login_access_sockets = 
@@ -64,7 +50,7 @@
 # Setting this to "no" means that Dovecot can be upgraded without
 # forcing existing client connections to close (although that could also be
 # a problem if the upgrade is e.g. because of a security fix).
-#shutdown_clients = yes
+shutdown_clients = yes
 
 # If non-zero, run mail commands via this many connections to doveadm server,
 # instead of running them directly in the same process.
