--- Makefile.am
+++ Makefile.am
@@ -11,7 +11,7 @@
 if ENABLE_EXAMPLES
   EXAMPLES      += examples
 endif
-SUBDIRS          = m4 po src $(EXAMPLES) tests doc
+SUBDIRS          = m4 po src
 
 ## Windows build files, for ZIP archive
 BORLAND_FILES         = cfgtest.bpf cfgtest.bpr config.h confuse.bpg libConfuse.bpf libConfuse.bpr
--- Makefile.in
+++ Makefile.in
@@ -393,7 +393,7 @@
 doc_DATA = README.md AUTHORS LICENSE ChangeLog.md
 EXTRA_DIST = README.md LICENSE ChangeLog.md libconfuse.spec.in libconfuse.spec
 EXAMPLES = $(am__append_1)
-SUBDIRS = m4 po src $(EXAMPLES) tests doc
+SUBDIRS = m4 po src
 BORLAND_FILES = cfgtest.bpf cfgtest.bpr config.h confuse.bpg libConfuse.bpf libConfuse.bpr
 DEVCPP_FILES = cfgtest.dev config.h libConfuse.dev
 MSVC6_FILES = config.h libConfuse.dsp libConfuse.dsw unistd.h
