--- configure
+++ configure
@@ -7010,7 +7010,7 @@
 
 
 
-for flag in -fPIE; do
+false && for flag in -fPIE; do
   as_CACHEVAR=`printf "%s\n" "ax_cv_check_cflags__$flag" | $as_tr_sh`
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking whether C compiler accepts $flag" >&5
 printf %s "checking whether C compiler accepts $flag... " >&6; }
@@ -7174,7 +7174,7 @@
 
 
 
-for flag in -fPIE -pie; do
+false && for flag in -fPIE -pie; do
   as_CACHEVAR=`printf "%s\n" "ax_cv_check_ldflags__$flag" | $as_tr_sh`
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking whether the linker accepts $flag" >&5
 printf %s "checking whether the linker accepts $flag... " >&6; }
