--- configure
+++ configure
@@ -15588,7 +15588,7 @@
 
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: **************************************** PTY device files" >&5
 printf "%s\n" "$as_me: **************************************** PTY device files" >&6;}
-if test "x$cross_compiling" = "xno"; then
+if true || test "x$cross_compiling" = "xno"; then
     as_ac_File=`printf "%s\n" "ac_cv_file_"/dev/ptmx"" | $as_tr_sh`
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for \"/dev/ptmx\"" >&5
 printf %s "checking for \"/dev/ptmx\"... " >&6; }
@@ -15647,7 +15647,7 @@
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: **************************************** entropy sources" >&5
 printf "%s\n" "$as_me: **************************************** entropy sources" >&6;}
 
-if test "x$cross_compiling" = "xno"; then
+if true || test "x$cross_compiling" = "xno"; then
 
 # Check whether --with-egd-socket was given.
 if test ${with_egd_socket+y}
