--- configure
+++ configure
@@ -19577,6 +19577,7 @@
 fi
      #  (Apps)
 #  Not-Used:
+if test "x$tryopenssl" != "xno"; then
 ac_fn_c_check_header_compile "$LINENO" "openssl/des.h" "ac_cv_header_openssl_des_h" "$ac_includes_default"
 if test "x$ac_cv_header_openssl_des_h" = xyes
 then :
@@ -19602,6 +19603,7 @@
 
 fi
 
+fi
 
 # RPM subdirectory path                   (Agent)
 if test "x$with_rpm" != "xno"; then
