--- libtransmission/session-settings.h
+++ libtransmission/session-settings.h
@@ -53,7 +53,7 @@
     V(TR_KEY_ratio_limit, ratio_limit, double, 2.0, "") \
     V(TR_KEY_ratio_limit_enabled, ratio_limit_enabled, bool, false, "") \
     V(TR_KEY_rename_partial_files, is_incomplete_file_naming_enabled, bool, false, "") \
-    V(TR_KEY_scrape_paused_torrents_enabled, should_scrape_paused_torrents, bool, true, "") \
+    V(TR_KEY_scrape_paused_torrents_enabled, should_scrape_paused_torrents, bool, false, "") \
     V(TR_KEY_script_torrent_added_enabled, script_torrent_added_enabled, bool, false, "") \
     V(TR_KEY_script_torrent_added_filename, script_torrent_added_filename, std::string, "", "") \
     V(TR_KEY_script_torrent_done_enabled, script_torrent_done_enabled, bool, false, "") \
